test/app.e2e-spec.ts

```diff
@@ -1,7 +1,7 @@
 import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
the `e2e` naming convention for tests is associated to UI testing (i.e. testing the end to end journey of an application). For our case, it feels like it's an api test overall. so the file name can be `markets.spec.ts` (without the `e2e` part)

Reply:
*.e2e-spec.ts naming convention is used in NestJs samples. *.spec.ts is used for unit tests, but these files are in different locations so we could use same suffix.
Should I still change to *.spec.ts?

Reply:
ok. let's use `*.api-test.ts` in that case. most likely we will need unit tests as well at some point